guide/principles/principles.md

```diff
@@ -0,0 +1,182 @@
+---
+layout: guide
+title: Principles
+description: The key principles to follow when designing Bitcoin products.
+nav_order: 9
+parent: Foundations
+permalink: /guide/foundations/principles/
+main_classes: -no-top-padding
+image: /assets/images/guide/foundations/principles/page-principles.jpg
+---
+
+<!--
+
+Editor's notes
+
+The key principles of designing for Bitcoin
+
+-->
+
+{% include picture.html
+   image = "/assets/images/guide/foundations/principles/principles.jpg"
+   retina = "/assets/images/guide/foundations/principles/principles@2x.jpg"
+   mobile = "/assets/images/guide/foundations/principles/principles-mobile.jpg"
+   mobileRetina = "/assets/images/foundations/principles/principles-mobile@2x.jpg"
+   alt-text = "Principles header illustration"
+   width = 1600
+   height = 600
+   layout = "full-width"
+%}
+
+# Bitcoin design principles
+
+As a new technology, Bitcoin offers the opportunity of a decentralized open financial system, where participants share the role of securing the network. This is important to give everyone equal and direct access without needing intermediaries. To make this a reality, we encourage everyone working on products to deliberately support the core principles of designing for Bitcoin. 
+
+These are principles we in the Bitcoin Design Community have identified and stand behind. Some of these come from the technology itself, and others from the behavior and ethos of the community. Although every use case and product is different, applications should strive to follow these principles. Diverging from them should only be done with very good reason.
+
+
+- [Self-custody](#self-custody)
+- [Security](#security)
+- [Inclusion](#inclusion)
+- [Interoperability](#interoperability)
+- [Transparency](#transparency)
+- [Privacy](#privacy)
+- [Decentralization](#decentralization)
+
+---
+
+## Self-custody
+
+> Let users control their private keys, with no risk for seizure or freezing of funds
+
+Our existing mental models of access to digital services are usernames and passwords, controlled by a company that has custody of your funds and data. With everyone having direct access to the Bitcoin network without middlemen, there is no longer a need for people to delegate control of their funds. While it comes with greater responsibility, it is self-custody that enables the open financial system of peer-to-peer transactions.
+
+**Do**
+- Let users control their private keys and funds directly
+- Create an easy path to self-custody for Bitcoin beginners
+
+**Don't**
+- Custody funds for your users
```

Comment:
```suggestion
- Custody funds for your users unless it cannot be avoided for your specific use case, and you have your users full consent
```

Reply:
Do you have examples of where this might be required? (other than centralized exchanges)
Not 100% opposed to including this but want to understand more.